File: src/main/scala/org/openapitools/server/model/WarehouseStockDTO.scala
package org.openapitools.server.model


/**
 * Информация об остатках товара.
 *
 * @param `type`  for example: ''null''
 * @param count Значение остатков. for example: ''null''
*/
final case class WarehouseStockDTO (
  `type`: WarehouseStockType,
  count: Long
)

File: ympa_fsharp_functions_server/src/model/WarehouseStockDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.WarehouseStockType

module WarehouseStockDTO =

  //#region WarehouseStockDTO

  [<CLIMutable>]
  type WarehouseStockDTO = {
    [<JsonProperty(PropertyName = "type")>]
    Type : WarehouseStockType;
    [<JsonProperty(PropertyName = "count")>]
    Count : int64;
  }

  //#endregion
